ITP/symmetry-breaking.tex

```diff
@@ -130,3 +130,15 @@
 \end{align*}
 This concludes the proof.
 \end{proof}
+
+We had quite some difficulty formalizing the symmetry-breaking argument.
+We spent many hours around a whiteboard,
```

Comment:
This line seems a bit unprofessional, I'm not sure we really need it. It seems better to characterize the difficulty as coming from the problem, with lines like "it is a delicate argument" or "it was challenging to find a formalization-friendly version of the reduction" without going into more details in the process itself.

Reply:
Agree with Mario